superbench/analyzer/file_handler.py

```diff
@@ -0,0 +1,154 @@
+        raw_data_df.to_excel(writer, sheet_name, index=True)
+    else:
+        logger.warning('DataDiagnosis: excel_data_output - {} data_df is empty.'.format(sheet_name))
+
+
+def excel_data_not_accept_output(writer, data_not_accept_df, rules):
+    """Output data_not_accept_df into 'Not Accept' excel page.
+
+    Args:
+        writer (xlsxwriter): xlsxwriter handle
+        data_not_accept_df (DataFrame): the DataFrame to output
+        rules (dict): the rules to diagnosis data
+    """
+    # Get the xlsxwriter workbook objects and init the format
+    workbook = writer.book
+    color_format_red = workbook.add_format({'bg_color': '#FFC7CE', 'font_color': '#9C0006'})
+    percent_format = workbook.add_format({'num_format': '0.00%'})
+
+    # Output the not accept
+    if isinstance(data_not_accept_df, pd.DataFrame):
+        data_not_accept_df.to_excel(writer, 'Not Accept', index=True)
+        if not data_not_accept_df.empty:
+            row_start = 1
+            row_end = max(row_start, len(data_not_accept_df))
+            columns = list(data_not_accept_df.columns)
+            worksheet = writer.sheets['Not Accept']
+
+            for rule in rules:
+                for metric in rules[rule]['metrics']:
+                    col_index = columns.index(metric)
+                    # Apply percent format for the columns whose rules are variance type.
+                    if rules[rule]['function'] == 'variance':
+                        worksheet.conditional_format(
+                            row_start,
+                            col_index,
+                            row_end,
+                            col_index,    # start_row, start_col, end_row, end_col
+                            {
+                                'type': 'no_blanks',
+                                'format': percent_format
+                            }
+                        )
+                    # Apply red format if the value violates the rule.
+                    if rules[rule]['function'] == 'value' or rules[rule]['function'] == 'variance':
+                        match = re.search(r'(>|<|<=|>=|==|!=)(.+)', rules[rule]['criteria'])
+                        if not match:
+                            continue
+                        symbol = match.group(1)
+                        condition = float(match.group(2))
+                        worksheet.conditional_format(
+                            row_start,
+                            col_index,
+                            row_end,
+                            col_index,    # start_row, start_col, end_row, end_col
+                            {
+                                'type': 'cell',
+                                'criteria': symbol,
+                                'value': condition,
+                                'format': color_format_red
+                            }
+                        )
+
+    else:
```

Comment:
Looks this else is pair of `if isinstance(data_not_accept_df, pd.DataFrame):`

Reply:
added new else for that